scss/custom/_application.scss

```diff
@@ -0,0 +1,55 @@
+.application {
```

Comment:
Can we consider renaming the component to "application-form" to be more precise.